.github/workflows/README.md

```diff
@@ -1,232 +1,665 @@
+    *   [1.3. Adding a feature](#GitHubActionsFunctionalRequirements-1.3.Addingafeature)
+    *   [1.4. Adding a hotfix](#GitHubActionsFunctionalRequirements-1.4.Addingahotfix)
+    *   [1.5. Manual Installer Build (Developer Testing)](#GitHubActionsFunctionalRequirements-1.5.ManualInstallerBuild(DeveloperTesting))
+*   [2\. List of the workflows](https://github.com/OpenBB-finance/OpenBBTerminal/tree/feature/improve-workflow-readme/.github/workflows#2-list-of-the-workflows)
+    *   [2.1. Branch Name Check](https://github.com/OpenBB-finance/OpenBBTerminal/tree/feature/improve-workflow-readme/.github/workflows#21-branch-name-check-done)
+    *   [2.2. Build & Release](#GitHubActionsFunctionalRequirements-2.2.Build&ReleasGreen)
+    *   [2.3. Build and Publish Docker](#GitHubActionsFunctionalRequirements-2.3.BuildandPublishDockeGreen)
+    *   [2.4. Release Drafter](#GitHubActionsFunctionalRequirements-2.4.ReleaseDrafteGreen)
+    *   [2.5. Deploy to GitHub Pages](#GitHubActionsFunctionalRequirements-2.5.DeploytoGitHubPageGreen)
+    *   [2.6. Integration Tests](#GitHubActionsFunctionalRequirements-2.6.IntegrationTests)
+    *   [2.7. General Linting](#GitHubActionsFunctionalRequirements-2.7.GeneralLintinGreen)
+    *   [2.8. MacOS M1 and Intel Build](#GitHubActionsFunctionalRequirements-2.8.MacOSM1andIntelBuilGreen)
+    *   [2.9. Mac OS X Full Clean Build with ML](#GitHubActionsFunctionalRequirements-2.9.MacOSXFullCleanBuildwithML)
+    *   [2.10. Nightly Build on develop branch](#GitHubActionsFunctionalRequirements-2.10.NightlyBuildondevelopbrancGreen)
+    *   [2.11. Deploy to PyPI - Nightly](#GitHubActionsFunctionalRequirements-2.11.DeploytoPyPI-NightlGreen)
+    *   [2.12. Deploy to PyPI](#GitHubActionsFunctionalRequirements-2.12.DeploytoPyPGreen)
+    *   [2.13. Unit Test](#GitHubActionsFunctionalRequirements-2.13.UnitTesGreen)
+    *   [2.14. Windows 10 Full Clean Build with ML](#GitHubActionsFunctionalRequirements-2.14.Windows10FullCleanBuildwithMGreen)
+    *   [2.15. Windows10 Build](#GitHubActionsFunctionalRequirements-2.15.Windows10BuilGreen)
+    *   [2.16. Reviewpad](#GitHubActionsFunctionalRequirements-2.16.Reviewpad)
+
+###  1\. OpenBB Terminal Github Processes
+------------------------------------
+
+### 1.1. Release Process
+
+**Purpose**  
+This process is a guide on how to build a new version of the OpenBB Installer.
+
+**Steps**
+
+*   The user will Create a branch for release from `develop` branch which follows the GitFlow naming convention i.e. `release/x.y.z`
+    
+*   The user will increment the version number in 3 files: `pyproject.toml` `setup.nsis` `feature_flags.py` and commit the changes
+    
+*   The [Build & Release]() workflow will be triggered on `push`. This will trigger the workflows to build the installers, publish the docker image and deploy to Pypi.
+    
+*   The User merges the `release` branch to the `main` and `develop` branch once the release process is completed successfully.
```

Comment:
Also need to do the release drafter to generate the new release on github.

Reply:
Noted